vars/checkPatch.groovy

```diff
@@ -25,7 +45,6 @@ def call(Map config) {
     checkout([
         $class: 'GitSCM',
         branches: [[name: branch]],
-        doGenerateSubmoduleConfigurations: false,
```

Comment:
Why did you need to remove this?

Reply:
Simplification.
https://issues.jenkins-ci.org/browse/JENKINS-43308